pkg/devspace/dependency/manager.go

```diff
@@ -39,7 +40,7 @@ func NewManagerWithParser(ctx devspacecontext.Context, configOptions *loader.Con
 
 type ResolveOptions struct {
 	SkipDependencies []string
-	Dependencies     []string
+	Dependencies     []string // Same as DependencyOptions.Only, either should be renamed
```

Comment:
The `devspace deploy --dependency` flag should be renamed `--only`, but thats beyond this pr and not backwards compatible. So i added this comment for clarity.

Reply:
Comment removed, need workflow approval (i think? not that familiar with these GitHub gates)